app/views/external_users/claims/supporting_documents/_summary.html.haml

```diff
@@ -44,9 +44,8 @@
           - if document.converted_preview_document.present?
             = link_to t('common.view'),
                       document_path(document),
-                      target: '_blank',
                       class: 'view',
-                      'aria-label': "View document: #{document.document_file_name}, Opens in a new window"
+                      'aria-label': "View document: #{document.document_file_name}"
```

Comment:
What's the advantage of removing the `opens in a new window` prompt here?

I understand that indiscriminately opening new windows is bad, but this informs the user before doing so.